Odata-docs/support/support-policy.md

```diff
@@ -0,0 +1,150 @@
+| 8.x | 2024-08-12 | 8.0.0 | 2024-08-12 | Active | TBD |
+| 7.x | 2016-08-22 |7.21.3 |2024-06-03 | Active | TBD |
+| 6.x | 2013-12-09 | 6.19.0 | 2017-11-14 | Maintenance Mode | 2025-07-15 |
+| 5.x | 2012-11-07 | 5.8.5 | 2021-11-12 | Maintenace Mode | TBD |
+
+### OData.NET 8.x
+
+OData.NET 8 is the next major release of the OData.NET libraries, including `Microsoft.OData.Core` 8, `Microsoft.OData.Edm` 8, `Microsoft.Spatial` 8 and `Microsoft.OData.Client` 8.
+These libraries will only support .NET 8 and later.
+
+OData.NET 8 will support the following OData protocal versions:
+
+- OData version 4.0
+- OData version 4.01
+
+### OData.NET 7.x
+
+OData.NET 7.x refers to 7.x versions of OData.NET libraries.
+
+The libraries are actively supported but will enter maintenance mode 6 months after the OData.NET 8 release.
+
+OData.NET 7 supports the following .NET versions:
+
+- .NET Core 3.1 and later
+- .NET Standard 1.1 and later
+- .NET Framework 4.5 and later
+
+OData.NET 7 supports the following OData protocol versions:
+
+- OData version 4.0
+- OData version 4.01
+
+### OData.NET 6.x
+
+OData.NET 6.x refers to 6.x versions of OData.NET libraries.
+
+The library is currently in maintenance mode and will go out of support on July 15, 2025.
+
+OData.NET 6 supports OData protocol version 4.0.
+
+### OData.NET 5.x
+
+OData.NET 5.x refers to 5.x versions of the following libraries:
+
+- Microsoft.Data.OData
+- Microsoft.Data.Edm
+- System.Spatial
+- Microsoft.Data.Services.Client
+- Microsoft.Data.Services
+
+The libraries are currently in maintenance mode. There are no current plans to end support. There will be an official announcements at least 12 months before official support ends.
+
+The libraries support .NET Framework 4.0 and later.
+
+The libraries support OData proctocol versions 1 to 3.
+
+## AspNetCore OData
+
+This refers to the `Microsoft.AspNetCore.OData`  library which adds OData integration to ASP.NET Core applications.
+
+| Version     | OData.NET version | Original release date | Latest patch version  | Latest patch release date | Support phase | End of support |
+| ------------|-------------------|-----------------------|-----------------------|---------------------------|---------------|----------------|
+| 9.x | 8.x | TBD | 9.0.0-rc.1 | 2024-06-05 | Preview | TBD |
+| 8.x | 7.x | 2021-07-03 | 8.2.5 |2023-03-01 | Active | TBD |
+| 7.x | 7.x | 2018-06-29 | 7.7.6 | 2024-07-12 | Active | TBD |
+
+Microsoft.AspNetCore.OData 9.x will be the next major release. This version will only support .NET 8 and later and OData.NET 8 and later.
+
+Microsoft.AspNetCore 8.x is currently actively supported. It supports .NET 5 and later and OData.NET 7.x.
+
+Microsoft.AspNetCore 7.x is currently actively supported but will enter maintenance mode soon. It supports .NET Framework 4.5 and later and .NET 5 and later. It supports OData.NET 7.x.
```

Comment:
Is this accurate? I don't think `Microsoft.AspNetCore 7.x` supports `.NET Framework 4.5`

Reply:
You're right, should .NET Standard 2.0 and .NET Core 3.1